web/packages/teleport/src/Navigation/Section.tsx

```diff
@@ -110,6 +113,9 @@ export function DefaultSection({
                 >
                   <subsection.icon size={16} />
                   <P2>{subsection.title}</P2>
+                  {subsection.isHyperLink && (
+                    <ArrowSquareOut size={16} color="text.muted" />
```

Comment:
Just confirming the vertical alignment on this. It maybe looks a little bit off in the screenshot?

Reply:
It's aligned, it just appears off due to an illusion caused by the shape not being uniform and having the arrow sticking out, the same thing can be seen in figma. Though I also just realized I used the wrong icon here 😄 

![image](https://github.com/user-attachments/assets/f7cb45a8-36db-42db-8c98-40c3737c7804)
![image](https://github.com/user-attachments/assets/5a07960e-18fb-49e7-be75-7edaa77343d3)